src/com/mambu/apisdk/services/AccountingService.java

```diff
@@ -89,17 +127,22 @@ public GLAccount getGLAccount(String glCode, String fromDate, String toDate) thr
 	 * @throws MambuApiException
 	 *             in case of an error
 	 */
-	public List<GLJournalEntry> getGLJournalEntries(Date fromDate, Date toDate) throws MambuApiException {

```

Comment:
Right.  I will add back methods with no "branchId" parameter and will deprecate them